### PR TITLE
Adding links to CIDs

### PR DESCRIPTION
**Pull request by Letme**
Now we link directly to Coverity server, so you can view the issue there

### CODE REVIEW

File: mlx/coverity.py [hunk 1]
```diff
@@ -274,8 +276,18 @@ def create_top_node(title):
     return top_node
 
 
-def create_cell(contents):
-    if isinstance(contents, str):
+def create_cell(contents, link=False):
```
Comment (SteinHeselmans) [inline mlx/coverity.py]: this `False` gives the impression it is a boolean, while it is a string, so i would change to `None` or `''`

Comment (SteinHeselmans) [reply mlx/coverity.py]: `url` is probably a better name for the `link` parameter

File: mlx/coverity.py [hunk 2]
```diff
@@ -274,8 +276,18 @@ def create_top_node(title):
...
+    if link is not False:
+        p_node = nodes.paragraph()
+        itemlink = nodes.reference()
+        itemlink['refuri'] = link
+        itemlink.append(nodes.Text(contents))
```
Comment (SteinHeselmans) [inline mlx/coverity.py]: in the  `elif` part below on lin 290 you check whether contents is a `str` object, that is not needed here?

File: mlx/coverity.py [hunk 3]
```diff
@@ -274,8 +276,18 @@ def create_top_node(title):
...
+        targetid = nodes.make_id(contents)
+        target = nodes.target('', '', ids=[targetid])
+        p_node += target
+        p_node += itemlink
+        return nodes.entry('', p_node)
```
Comment (SteinHeselmans) [inline mlx/coverity.py]: no return but `contents = p_node` here, analog to the elif case?

Comment (Letme) [reply mlx/coverity.py]: Yeah, it was bad indeed. I did a bit of refactor to make it more obvious.

File: mlx/coverity.py (hunk 1 above)
Comment (Letme) [reply mlx/coverity.py]: url is indeed clearer and None as well. Fixed it in next commit

**Review by SteinHeselmans — APPROVED** [review]

**Pull request merged**